### DISCUSSION TITLE
Elsa.EntityFrameworkCore not included in nuget package

### DISCUSSION THREAD

**Discussion by vinujosey** [elsa-workflows/elsa-core#5602] [General]
Hi,

I am trying to setup a sample workflow using Elsa 3.13. But when I download the NuGet package the references for  Elsa.EntityFrameworkCore.* is not included in the package. Is there anything specific that I need to to inorder to get that?

**Comment by glime-ai** [comment]
To set up a sample workflow using Elsa 3.13 and include the `Elsa.EntityFrameworkCore.*` references, you need to manually install the necessary Entity Framework Core packages for Elsa. These packages are not included by default when you install the main Elsa package. You can add them using the .NET CLI or through your IDE's NuGet package manager.

Here are the steps to install the required packages using the .NET CLI:

1. Open your command line interface.
2. Navigate to the directory of your project.
3. Run the following commands to install the Elsa Entity Framework Core packages:

```bash
dotnet add package Elsa.EntityFrameworkCore --version 3.13.0
dotnet add package Elsa.Persistence.EntityFrameworkCore --version 3.13.0
```

Please replace `3.13.0` with the actual version number you are using if it's different. Also, ensure that you have the correct version of the .NET SDK installed and that your project targets a compatible framework version.

If you are using a specific database provider (e.g., SQL Server, SQLite), you may also need to install the corresponding Entity Framework Core database provider package. For example:

```bash
dotnet add package Elsa.Persistence.EntityFrameworkCore.SqlServer --version 3.13.0
```

or for SQLite:

```bash
dotnet add package Elsa.Persistence.EntityFrameworkCore.Sqlite --version 3.13.0
```

After installing the necessary packages, you should be able to use the Entity Framework Core features with Elsa in your project.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Reply by vinujosey** [reply]
Thanks !!
